server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -345,7 +345,18 @@ private CacheInfo getCacheInfo(RestRequest request, EmbeddedCacheManager cacheMa
             Cache<?, ?> cache = restCacheManager.getCache(cacheName, request);
             cacheInfo.status = cache.getStatus().toString();
          } else {
-            cacheInfo.status = ComponentStatus.FAILED.toString();
+            switch (cacheHealth) {
```

Comment:
This isn't quite right, the if statement above prevents this from ever being anything other than FAILED afai can see.

Reply:
Once again, back copy of code :/ Should be ok now. Thanks.